src/Components/Main/Main.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import { useTodos, useTodosDispatch } from '../TodoContext/TodoContext';
+import { TodoList } from '../TodoList/TodoList';
+import { State } from '../../types/State';
+
+export const Main: React.FC = () => {
+  const todos = useTodos();
+  const dispatch = useTodosDispatch();
+  const [toggleAll, setToggleAll] = useState(false);
```

Comment:
Rename all boolean variables according [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
```suggestion
  const [isToggleAll, setIsToggleAll] = useState(false);
```